src/lib/pageVideoStreamWriter.ts

```diff
@@ -172,16 +172,17 @@ export default class PageVideoStreamWriter extends EventEmitter {
       source: this.videoMediatorStream,
       priority: 20,
     })
-      .videoCodec('libx264')
+      .videoCodec(this.options.videoCodec || 'libx264')
       .size(this.videoFrameSize)
       .aspect(this.options.aspectRatio || '4:3')
       .autopad(this.autopad.activation, this.autopad?.color)
       .inputFormat('image2pipe')
       .inputFPS(this.options.fps)
-      .outputOptions('-preset ultrafast')
-      .outputOptions('-pix_fmt yuv420p')
-      .outputOptions('-minrate 1000')
-      .outputOptions('-maxrate 1000')
+      .outputOptions(`-crf ${this.options.videoCrf ?? 23}`)
```

Comment:
Could you please add a bit of description stating why it is defaults to 23 ?

Reply:
Could quote this page? https://trac.ffmpeg.org/wiki/Encode/H.264  "23 is the default"